app/activity_notifications/chat_message_sent.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class ChatMessageSent < Noticed::Base
+  deliver_by :database, format: :to_database, association: :activity_notifications
+
+  param :contributor_id, :request_id, :user_id, :message_id
+
+  def to_database
+    {
+      type: self.class.name,
+      contributor_id: params[:contributor_id],
+      request_id: params[:request_id],
+      user_id: params[:user_id],
+      message_id: params[:message_id]
+    }
+  end
+
+  def group_key
+    [record.request_id, record.user_id]
+  end
+
+  def record_for_avatar
+    record.user
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def group_message(notifications:)
+    unique_contributors = notifications.map(&:contributor).uniq
+    count = unique_contributors.size
+
+    t(group_message_key(notifications.first.recipient_id, count),
+      contributor_one: unique_contributors.first.name,
+      contributor_two: unique_contributors.second&.name,
+      request_title: record.request.title,
+      user_name: record.user.name,
+      count: count,
+      others_count: count - 1).html_safe
+  end
+  # rubocop:enable Rails/OutputSafety
+
+  def url
+    request_path(record.request, anchor: "message-#{record.message.id}")
+  end
+
+  def link_text
+    t('.link_text')
+  end
+
+  def pluralization_key(count)
+    case count
+    when 1
+      :one
+    when 2
+      :two
+    else
+      :other
+    end
+  end
```

Comment:
There is duplication with this method and the one in `MessageReceived` class. I considered if it was worth it to add these to a helper, but as it only appears twice, didn't think it was worth it. If the events grow and we want to maintain different German for one, two, and other than I think it would be worth it. What do others think?